javalib/src/main/scala/java/lang/System.scala

```diff
@@ -173,6 +159,41 @@ object System {
     }
   }
 
+  private def getUserLocaleInfo(
+      infoCode: LCType,
+      bufSize: UInt
+  ): Option[String] = {
+    val buf = stackalloc[CChar16](bufSize)
+    GetLocaleInfoEx(
+      LOCALE_NAME_USER_DEFAULT,
+      infoCode,
+      buf,
+      bufSize
+    ) match {
+      case 0 => None
+      case _ => Some(fromCWideString(buf, StandardCharsets.UTF_16))
+    }
+  }
+  private def getUserLanguage(): Option[String] = {
+    if (isWindows) {
+      getUserLocaleInfo(LOCALE_SISO639LANGNAME2, 9.toUInt)
```

Comment:
The 9 is super obscure. Consider
```suggestion
      getUserLocaleInfo(LOCALE_SISO639LANGNAME2, bufSize = 9.toUInt)
```